### PR TITLE
Change copyright text prop-type

### PR DESCRIPTION
**Pull request by EinsteinNjoroge**
Relates to [PR934](https://github.com/bbc/psammead/issues/934)

**Context:**
_We need to add a lang attribute to the copyright symbol so that it is always read in English._

**Overall change:** _Change copyright text proptype from string text to node._

**Code changes:**

- _Change copyright text proptype from string text to node._

---

- [X] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-sitewide-links/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ npm install @bbc/psammead-sitewide-links --save
 | Argument      | Type                  | Required | Default | Example                                           |
 | ------------- | --------------------- | -------- | ------- | ------------------------------------------------- |
 | links         | Array of Link objects | Yes      | N/A     | `[{href:'https://www.bbc.com', text: 'The BBC'}]` |
-| copyrightText | String                | Yes      | N/A     | `'Copyright BBC News'`                            |
+| copyrightText | Node                | Yes      | N/A     | `<span>Copyright BBC News</span>`                            |
```
Comment (ghost) [inline packages/components/psammead-sitewide-links/README.md]: include the `copyright` symbol in the example

**Review by ghost — DISMISSED** [review]

File: packages/components/psammead-sitewide-links/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 3.0.0 | [PR#1524](https://github.com/bbc/psammead/pull/1524) Change `copyrighText` prop from `sting` to `node`  |
```
Comment (DenisHdz) [inline packages/components/psammead-sitewide-links/CHANGELOG.md]:
```suggestion
| 3.0.0 | [PR#1524](https://github.com/bbc/psammead/pull/1524) Change `copyrighText` prop from `string` to `node`  |
```

**Review by DenisHdz — CHANGES_REQUESTED** [review]

File: packages/components/psammead-sitewide-links/README.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ const props = {
     { href: 'https://www.bbc.co.uk/news', text: 'BBC News' },
     { href: 'https://www.bbc.co.uk/sport', text: 'BBC Sport' },
   ],
-  copyrightText: 'Copyright BBC News',
+  copyrightText: <span>@ BBC News<span>,
```
Comment (j-pendlebury) [inline packages/components/psammead-sitewide-links/README.md]:
```suggestion
  copyrightText: <span>@ BBC News</span>,
```

**Review by j-pendlebury — COMMENTED** [review]
Just one thing 🙈

**Review by DenisHdz — APPROVED** [review]
LGTM

**Review by Bopchy — APPROVED** [review]

**Review by j-pendlebury — APPROVED** [review]

**Comment by staylos92** [comment]
This doesn't need testing by a tester

**Pull request merged**